app/bin/databricks_const.py

```diff
@@ -20,6 +24,11 @@
 
 # Command execution configs
 COMMAND_TIMEOUT_IN_SECONDS = 300
-COMMAND_SLEEP_INTERVAL_IN_SECONDS = 10
+COMMAND_SLEEP_INTERVAL_IN_SECONDS = 3
+
+USER_AGENT_CONST = "Databricks-AddOn/1.1.0 Splunk/{}"
```

Comment:
it's not allowed to change user agent prefix from `Databricks-AddOnFor-Splunk-`, as internal telemetry depends on it. you can leave ` Splunk/{}` after version sequence number. 

and stability-wise, it's probably 0.2.0 and no way 1.1.0